sdk/core/azure-core/test/nlohmann-json-test/thirdparty/doctest/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016-2019 Viktor Kirilov
+
```

Comment:
I think we may need a dev-time notices and  cgmanifest update related to doctest as another unit test framework used within the SDK.